WebServer/frontend/src/app/video-utils/video-utils.component.ts

```diff
@@ -0,0 +1,60 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { RecordService } from "../_services/record.service";
+import { FileuploadService } from "../_services/fileupload.service"
+import { AuthService } from '../_services/auth.service';
+import { IMediaStream } from "../_services/streamMessage.service";
+
+@Component({
+  selector: 'app-video-utils',
+  templateUrl: './video-utils.component.html',
+  styleUrls: ['./video-utils.component.scss']
+})
+export class VideoUtilsComponent implements OnInit {
+  @Input() stream: IMediaStream;
+  fileToUpload: File = null;
+  canUpload: Boolean = false;
+
+  constructor(
+    private recordService: RecordService, 
+    private fileUploadService: FileuploadService,
+    private auth: AuthService,
+    ) { }
+
+  ngOnInit(): void {
+  }
+
+  handleFileInput(files: FileList) {
+    this.fileToUpload = files.item(0);
+    this.canUpload = true;
+  }
```

Comment:
Hvad hvis man har givet en fil og så fjerner den? Min vurdering er, at denne metode kaldes, files.item(0) tilgår null, men canUpload forbliver true.

Reply:
Fixed it